package.json

```diff
@@ -146,6 +147,7 @@
   "peerDependencies": {
     "@emotion/core": "10.0.35",
     "@emotion/styled": "10.0.27",
+    "emotion-theming": "10.0.27",
```

Comment:
We've run into compatibility issues in the past in applications that use our library if there's an `@emotion` version mismatch, see #323. Basically, styles don't always apply as expected, which resulted in (mostly) padding/alignment issues. 

It's higher maintenance to handle it this way but I think it is worth the small cost.